server/common/database/helpers.js

```diff
@@ -1,14 +1,11 @@
 'use strict';
 
-const { Sequelize, Op } = require('sequelize');
+const { Op, Sequelize } = require('sequelize');
 const has = require('lodash/has');
 const inRange = require('lodash/inRange');
 const last = require('lodash/last');
 
-// eslint-disable-next-line no-extra-parens
-const AsyncFunction = (async function () {}).constructor;
-
-const isAsyncFunction = arg => arg instanceof AsyncFunction;
```

Comment:
Now we wrap all methods, not only those declared as `async` (Same implementation as in [tailor](https://github.com/ExtensionEngine/tailor)).